helix-term/src/commands.rs

```diff
@@ -319,6 +319,7 @@ impl MappableCommand {
         redo, "Redo change",
         earlier, "Move backward in history",
         later, "Move forward in history",
+        add_undo_checkpoint, "Add undo checkpoint",
```

Comment:
I feel like `commit_undo_checkpoint` might read slightly better.

Reply:
or even commit_undo_state, but I figured calling it commit would be confusing if git integration was added in the future

Reply:
I think diff and possibly blame are likely to end up in the core but any sort of git commit integration would probably live as a plugin. Since we already use "commit" when talking about history in the implementation I think `commit_undo_checkpoint` is reasonable